src/dst/bind.cpp

```diff
@@ -882,18 +882,24 @@ static std::unique_ptr<Expr> fracture(Top &top, bool anon, const std::string &na
     // don't fail if unbound; leave that for the second pass
     rebind_ref(binding, ref->name, ref->fragment);
     return expr;
-  } else if (expr->type == &Subscribe::type) {
+  }
```

Comment:
Why change this at all?

Reply:
ah, it was an unrelated cleanup. Was trying to figure out what the code did and this was de-cluttering so I could think better/understand what was going on